config/settings.py

```diff
@@ -130,20 +130,20 @@
 
 
 # Check https://auth0.com/docs/configure/apis/scopes for more info
-SOCIAL_AUTH_AUTH0_SCOPE = ["openid", "profile", "email"]
+SOCIAL_AUTH_IDP_SCOPE = ["openid", "profile", "email"]
```

Comment:
This might break things. Check this documentation: https://auth0.com/docs/quickstart/webapp/django/01-login

Reply:
@iamjazzar what are you doing here 😅, okay thanks!!! I'll fix it.